app/routes/circulars.md

```diff
@@ -5,16 +5,123 @@ meta:
 
 # GCN Circulars
 
-- [Real-time archive of all GCN Circulars by date](https://gcn.gsfc.nasa.gov/gcn3_archive.html)
-- [GCN Circulars by object](https://gcn.gsfc.nasa.gov/selected.html)
+GCN Circulars are rapid astronomical bulletins submitted by and distributed to community members worldwide.
+
+- Circulars are **brief**, **citeable**, and **human-readable**.
+- Circulars can announce observations, quantitative near-term predictions, requests for follow-up observations, or future observing plans.
+
+Circulars allow the GRB community to **email** messages to a central queue where they are automatically vetted and distributed to the entire GRB community. These prose-style messages allow the community to coordinate and make optimum use of its limited resources. 
+
+The GCN Circulars list is independent from the machine-readable [GCN Notices (TODO:link)](notices.md) system.
+
+
+ > **Quick [submission](#circular-submission) refresher**
+ > 1. Email your submission to gcncirc@capella2.gsfc.nasa.gov.
+ > 2. The email subject-line must contain at least one of the strings in the [list of instruments and source types](#subject-word-list).
```

Comment:
Do you intend this to be a [summary box](https://designsystem.digital.gov/components/summary-box/)? If so, I can format that for you later.

Reply:
Yes, that looks good. Thanks.